htdocs/map.html

```diff
@@ -6,14 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <meta name="theme-color" content="#222" />
     <link rel="shortcut icon" type="image/x-icon" href="static/favicon.ico" />
-    <script src="compiled/map.js"></script>
+    <script src="compiled/mapLeaflet.js"></script>
```

Comment:
We would need to have a setting in the admin interface to select which maps should be used by the map interface.
I'll leave this to you, since I will take more time for me to research how to do this.
The setting should change only the script included here.